src/neo-vm/ExecutionEngine.cs

```diff
@@ -22,7 +22,7 @@ public class ExecutionEngine : IDisposable
         /// <summary>
         /// Min value for SHL and SHR
         /// </summary>
-        public virtual int Min_SHL_SHR => -256;
+        public virtual int Min_SHL_SHR => 0;
```

Comment:
We can change both variables to short, and we should check if is a breaking change

Reply:
It's not a breaking change, @vncoelho will make this sure for us ;)
`short` or `int` is the same here.